README.md

```diff
@@ -64,9 +64,9 @@ rm index-robust04-20191213.tar.gz
 Use the `SimpleSearcher` for searching:
 
 ```python
-from pyserini.search import pysearch
+from pyserini.search import SimpleSearcher
```

Comment:
As for the readme.md examples, I think it should reflect more on what will be the most common use cases:
1- How to build an index
2- How to query the index
3- How to retrieve a document (contents, fields) given its id

The other examples (e.g. Usage of the Analyzer API) can be kept, but I think most users only care about those first 3.

Reply:
Yes, will do this as part of #119 later?

Reply:
SG!